exasol/slc/api/run_db_tests.py

```diff
@@ -103,6 +104,7 @@ def run_db_test(
     log_level: Optional[str] = None,
     use_job_specific_log_file: bool = True,
     compression_strategy: CompressionStrategy = defaultCompressionStrategy(),
+    accelerator: Accelerator = Accelerator.NONE,
```

Comment:
not sure if this is a consistency point, but would you want Accelerator.None here to use `defaultAccelerator`?